src/button.ts

```diff
@@ -39,7 +39,7 @@ function init() {
   if (activateManager || window.localStorage.getItem('collect-manager') === 'true') {
     window.localStorage.setItem('collect-manager', 'true');
     if (activateManager) {
-      window.location.replace(window.location.href.replace('?collectManager=1', ''));
+      window.location.replace(window.location.href.replace(`?collectManager${activateManager === 'undefined' ? '' : `=${activateManager}`}`, ''));
```

Comment:
What if they have ?foo=1&collectManager=1. Does that leading ? in the replace cause an infinite loop?

Reply:
just updated it

Reply:
Use this:
`window.location.replace(window.location.href.replace(/collectManager[^&]*/g, ''))`